src/pages/groupComparison/GroupComparisonUtils.tsx

```diff
@@ -250,21 +250,20 @@ export function getStudyIds(groups: Pick<SessionGroupData, 'studies'>[]) {
 export function getSampleIdentifiers(
     groups: Pick<SessionGroupData, 'studies'>[]
 ) {
-    return _.uniqWith(
-        _.flattenDeep<SampleIdentifier>(
-            groups.map(group =>
-                group.studies.map(study => {
-                    const studyId = study.id;
-                    return study.samples.map(sampleId => ({
-                        studyId,
-                        sampleId,
-                    }));
-                })
-            )
-        ),
-        (id1, id2) =>
-            id1.sampleId === id2.sampleId && id1.studyId === id2.studyId
-    );
+    const sampleIds: { [key: string]: SampleIdentifier } = {};
+
+    groups.forEach(group => {
+        group.studies.forEach(study => {
+            study.samples.forEach(sample => {
+                sampleIds[study.id + '\n' + sample] = {
```

Comment:
why linebreak instead of say, underscore?  just curious.  seems strange but i get that it isn't used.

Reply:
Really, we could use any character that we exclude from study, sample and patient ids. I chose to use `\n` because I know that if that character was included in an id, the study would fail to import. I was concerned that other characters may technically be excluded according to our specifications, but that we may not actually enforce that during import.